### PR TITLE
Feature/map

### PR DESCRIPTION
**Pull request by leighlars**
# Description
This PR adds a new component `About` to the site. The `About` page renders info about the purpose, dev info, legend of icons, etc. This has been styled and tested. #15 and #14 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [X] No
- [X] Yes

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
I checked to render the components, but did not test the fired event in Header.
# Message for reviewer:
Start in `About` and run `npm test`

# Checklist:
- [X] My code has no unused/commented out code
- [X] My code has no console logs
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/About/About.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import './About.scss'
+import React, { useState, useEffect } from "react";
+import aqtable from '../assets/ap3.jpg'
+
+const About: React.FC = () => {
+  return (
+   <section className="about-section">
```
Comment (ErinUntermeyer) [inline src/About/About.tsx:7]: This page looks AWESOME! Great idea, I think it's a wonderful addition to our page!

File: src/Location/Location.tsx [hunk 2]
```diff
@@ -4,10 +4,7 @@ import './Location.scss'
 const Location: React.FC = () => {
 	return (
 		<section className="location-data">
-			<p>City, State</p>
-			<p>Country</p>
-			<p>US AQI: </p>
-			<p>🌪</p>
+			<p></p>
```
Comment (ErinUntermeyer) [inline src/Location/Location.tsx:7]: Thanks for removing this! I didn't mean to leave it in. Once I get some data fetched from this new weather API, I will add some p tags back in to hold and display the data. I'd love to get your tips and tricks on styling so it looks consistent with the rest of the app!

File: src/About/About.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    <img
+     src={aqtable}
+     className="aq-image"
+     alt="table of Air Quality information from EPA.gov"
+    />
+    <span className="credits">
+     <p>App Developed by:</p>
+     <br/>
+     <a href="https://github.com/ErinUntermeyer">Erin Untermeyer</a>
+     <a href="https://github.com/JoshSevy">Josh Sevy</a>
```
Comment (JoshSevy) [inline src/About/About.tsx:27]: Awesome, we are famous!

File: src/Header/Header.tsx [hunk 4]
```diff
@@ -15,6 +17,21 @@ const Header: React.FC = () => {
         <h2 className='tagline'><i>Plan. Research. Experience.</i></h2>
       </div>
       <Search />
+      </div>
+      <nav>
+        <NavLink to='/'
```
Comment (JoshSevy) [inline src/Header/Header.tsx:22]: All the code looks great, great work!

**Pull request merged**